src/src/com/microsoft/aad/adal/AuthenticationActivity.java

```diff
@@ -886,127 +867,98 @@ private void appendAppUIDToAccount(StorageHelper cryptoHelper, Account account)
             }
         }
 
-        private void setAccount(final TokenTaskResult result) throws InvalidKeyException,
-                InvalidKeySpecException, InvalidAlgorithmParameterException,
-                IllegalBlockSizeException, BadPaddingException, IOException {
+        private void setAccount(final TokenTaskResult result)
+                throws GeneralSecurityException, UnsupportedEncodingException {
             // TODO Add token logging
             // TODO update for new cache logic
 
             // Authenticator sets the account here and stores the tokens.
-            try {
```

Comment:
I just removed try catch block from here...but this diff tool can't analyze it...